src/sage/rings/lazy_series.py

```diff
@@ -584,7 +584,7 @@ def map_coefficients(self, f):
 
     def truncate(self, d):
         r"""
-        Return this series with its terms of degree >= ``d`` truncated.
+        Return this series with its terms of degree `\geq` ``d`` truncated.
```

Comment:
I think it would be clearer to write: `Return the series obtained by removing all terms of degree at least ``d``.`

Reply:
I agree! I'll change it.